cmd/ebrelayer/events/event.go

```diff
@@ -47,14 +58,17 @@ func NewLockEvent(contractAbi abi.ABI, eventName string, eventData []byte) LockE
 // PrintEvent : prints a LockEvent struct's information
 func PrintEvent(event LockEvent) {
 	// Convert the variables into a printable format
+	chain := event.Chain
+	contract := event.Contract
```

Comment:
rename ChainID and ContractAddress

Reply:
https://github.com/swishlabsco/peggy/pull/4/commits/be6ce9d49dac7195bbcf2bf3b5260ebd2afc5715